### PR TITLE
fix(core): :bug: inputs/CoreFormControl fix valid props

### PR DESCRIPTION
**Pull request by anantakumarghosh**
## Description

fix valid props CoreFormControl
#354
## Checklist

- [x] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [x] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Reviewers

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

File: package/components/inputs/CoreFormControl.js [hunk 1]
```diff
@@ -40,34 +40,34 @@ CoreFormControl.validProps = [
...
   },
   {
     name : "focused",
     types: [{ type: "boolean" }]
   },
   {
-    name : "fulwidth",
-    types: [{ default: false, type: "boolean" }]
+    name : "fullWidth",
+    types: [{ default: false, type: "boolean", validValues: [false, true] }]
```
Comment (techoneel) [inline package/components/inputs/CoreFormControl.js:55]: https://github.com/wrappid/native-web/commit/2b7cd42fa5664a0e4029cfdd7c32c414d597c5df

Fullwidth default supposed to be true or false

**Review by techoneel — CHANGES_REQUESTED** [review]
Raising concern

File: package/components/inputs/CoreFormControl.js (hunk 1 above)
Comment (anantakumarghosh) [reply package/components/inputs/CoreFormControl.js:55]: `fullWidth` should be `false` by default. Wherever required, true should be passed as props.

**Review by techoneel — APPROVED** [review]
Good to go

**Pull request merged**